src/components/UniversalNavbarExpanded/UniversalNavbarExpanded.module.scss

```diff
@@ -95,3 +95,14 @@ $laptop-range-small-end: 929px;
     display: none;
   }
 }
+
+.estimateCopyOptimizely {
```

Comment:
Unrelated, but have we decided to use camel case instead of pascal case? I've still been trying to use pascal case but more than happy to switch to camel case if we've decided we want that going forward 👍

Reply:
I'm not sure if we've documented a standard for EDS regarding casing. We should if we haven't. Good catch.